src/main/java/mate/controller/car/AddDriverToCarController.java

```diff
@@ -12,6 +12,7 @@
 import mate.service.DriverService;
 
 public class AddDriverToCarController extends HttpServlet {
+    public static final String DRIVER_ID = "driver_id";
```

Comment:
Parameter "driver_id" from request and parameter "driver_id" from session it's different parameters, so lets leave request parameter as it was